src/Compilers/Server/VBCSCompilerTests/ClientConnectionTests.cs

```diff
@@ -228,7 +228,7 @@ private static TestableClientConnection CreateConnection(Stream stream, ICompile
             Assert.Equal(BuildResponse.ResponseType.Rejected, response.Type);
         }
 
-        [Fact]
+        [ConditionalFact(typeof(WindowsOnly))]
```

Comment:
Why `WindowsOnly` rather than `ClrOnly`?

Reply:
I guess the title of this PR is a bit misleading. I'm going off of Jared's advice to make the CoreCLR+Windows compiler server work first, and only then make CoreCLR+crossplat work. There's a lot of additional work to get e.g. IPC working on non-Windows platforms.

(ClrOnly means "don't run this on Mono", which isn't too relevant here - I assume you meant it as "run CoreClr on linux/mac too")